src/views/preview/comment/comment.jsx

```diff
@@ -71,15 +112,19 @@ class Comment extends React.Component {
                                     Delete {/* TODO internationalize */}
                                 </span>
                             ) : null}
-                            <span className="comment-report">
+                            <span
+                                className="comment-report"
+                                onClick={this.handleReport}
+                            >
                                 Report {/* TODO internationalize */}
                             </span>
                         </div>
                     </FlexRow>
                     <div
                         className={classNames({
                             'comment-bubble': true,
-                            'comment-bubble-deleted': deleted
+                            'comment-bubble-deleted': deleted,
+                            'comment-bubble-reported': reported
                         })}
                     >
                         {/* TODO: at the moment, comment content does not properly display
```

Comment:
Should also localize 'reply' (At line 145) - I can't leave a comment on it as it's not in the diffs.

Reply:
Fixed in c19adee